rootfs/standard/usr/bin/mynode_update_latest_version_files.sh

```diff
@@ -31,4 +31,5 @@ echo $SPHINXRELAY_VERSION > $SPHINXRELAY_LATEST_VERSION_FILE
 echo $NETDATA_VERSION > $NETDATA_LATEST_VERSION_FILE
 echo $WEBSSH2_VERSION > $WEBSSH2_LATEST_VERSION_FILE
 echo $PYBLOCK_VERSION > $PYBLOCK_LATEST_VERSION_FILE
-echo $WARDENTERMINAL_VERSION > $WARDENTERMINAL_LATEST_VERSION_FILE
\ No newline at end of file
+echo $WARDENTERMINAL_VERSION > $WARDENTERMINAL_LATEST_VERSION_FILE
+echo $ALBYHUB_VERSION > $ALBYHUB_LATEST_VERSION_FILE
```

Comment:
This can be removed. Dynamic apps don't need to go here. I should add a comment in this file.

Reply:
ok. Removed addition that is not needed.